tests/test_MocLogin.py

```diff
@@ -0,0 +1,72 @@
+from django.test import TestCase
+from core.models import AtmosphereUser as User
+from core.models import Provider
+from core.models import PlatformType
+from core.models import ProviderType
+
+from threepio import logger
+
+from atmosphere import settings
+
+
+# This is assuming a clean database
+class MocLogin(TestCase):
+    username = 'friday-test'
+    password = 'friday-test'
```

Comment:
I don't think it is a good idea of putting the password there in a public repo. Also, what if this test user no long exist? Maybe a more generic way of testing this.